Example/Tests/ServerConfigurationTests.swift

```diff
@@ -51,4 +53,30 @@ class ServerConfigurationTests: XCTestCase {
         let url = dictionaryToUrlUnencodedParams(dict: ["page": 1, "key": "api%20key"])
         XCTAssert(url == "key=api%20key&page=1" || url == "page=1&key=api%20key")
     }
+    
+    func testApplyCookiePolicy() {
```

Comment:
Also make sure to add one to make sure that allowing cookies doesn't remove the cookies